src/auth/header-validation.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@nestjs/common';
+import { CalendarEventAuthor } from '../calendar/interfaces/calendar.interface';
+import { DiscordProfileAuthor } from '../discord/interfaces/discord.interface';
+import { StandupAuthor } from '../standup/interfaces/standup.interface';
+
+@Injectable()
+export class ValidationService {
+  public validateAuthor(
+    objectAuthor: CalendarEventAuthor | DiscordProfileAuthor | StandupAuthor,
```

Comment:
Rather in importing other modules to here, would it be cleaner to have an interface in this module, that the other modules inherit - that way this module doesn't need them, but the other modules will need the auth module, so it is one way rather than 2 ways (I hope that makes sense 😂 )